### ISSUE TITLE
Bug: prop name is `allowFullScreen`

### ISSUE THREAD

**Issue by moltar** [robbestad/react-iframe#123]
bcae7bf6744fcb2f10f3bfca0b2c7f7bc2238f74 introduced a bug

```
  ● Console

    console.error
      Warning: Invalid DOM property `allowFullscreen`. Did you mean `allowFullScreen`?
```

**Comment by robbestad** [comment]
Should be fixed :)

**Comment by natoszme** [comment]
Hi @robbestad is it supposed to be fixed on v1.8.4?

Thanks!
